docker-compose.yml

```diff
@@ -6,6 +6,10 @@ services:
     build:
       context: mat-process-api/
       dockerfile: ./Dockerfile
+    environment:
+        - MONGO_CONN_STRING=mongodb://mongo-db:27017/?gssapiServiceName=mongodb
+    links:
+      - mongo-db
```

Comment:
Why did the docker file need updating?

Reply:
I committed this by mistake, will remove it.